frontend/packages/employee-frontend/src/components/fee-decisions/FeeDecisionFilters.tsx

```diff
@@ -43,6 +52,10 @@ function FeeDecisionFilters() {
     void getUnits([], 'DAYCARE').then(setUnits)
   }, [])
 
+  useEffect(() => {
+    void getEmployees().then(setEmployees)
```

Comment:
You should not use a list of all employees in the filter, it's going to be very long and most of the values are irrelevant. You'll have to create a separate endpoint for that.